src/test/java/io/reactivex/internal/operators/flowable/FlowableWithLatestFromTest.java

```diff
@@ -261,7 +264,10 @@ public void testNoDownstreamUnsubscribe() {
 
     @Test
     public void testBackpressure() {
-        Flowable<Integer> source = Flowable.range(1, 10);
+        TestScheduler testScheduler = new TestScheduler();
```

Comment:
ok, I have added a test, but I still had to modify testBackpressure because it was relying on the fact that the source would not get more request if the other stream does not emit data.